types.d.ts

```diff
@@ -11,15 +11,17 @@ declare module 'tailwindcss/src/lib/generateRules.js' {
 }
 
 declare module 'tailwindcss/src/lib/setupContextUtils.js' {
-  import { TailwindConfig } from 'tailwindcss/tailwind-config';
   import { Container, Root } from 'postcss';
+  import { TailwindConfig } from 'tailwindcss/tailwind-config';
+
+  type ModifierFunction = (options: { className?: string; selector: string }) => string;
```

Comment:
where the hell did you found that?!!! xD

Reply:
I based it purely on the implementation in `getVariants`. There’s a good chance I missed several optional properties.

Reply:
upps ^^ you just put in there the object from the code ^^